CondFormats/CTPPSReadoutObjects/xml/rpix_channel_mask_220_far.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<top>
+  <arm id="0">
+    <station id="2">
+      <!--4a -->
+      <!--45 220 hor far-->
+      <rp_detector_set id="3" doCorrelations="0">
+        <rpix_plane id="0">
+        </rpix_plane>
+        <rpix_plane id="1"></rpix_plane>
+        <rpix_plane id="2"></rpix_plane>
+        <rpix_plane id="3"></rpix_plane>
+        <rpix_plane id="4"></rpix_plane>
+        <rpix_plane id="5"></rpix_plane>
+      </rp_detector_set>
+    </station>
+  </arm>
+  <arm id="1">
+    <station id="2">
+      <!--56 220 hor far-->
+      <rp_detector_set id="3" doCorrelations="0">
+        <rpix_plane id="0"></rpix_plane>
+        <rpix_plane id="1"></rpix_plane>
+        <rpix_plane id="2"></rpix_plane>
+        <rpix_plane id="3"></rpix_plane>
+        <rpix_plane id="4"></rpix_plane>
+        <rpix_plane id="5"></rpix_plane>
+      </rp_detector_set>
+    </station>
+  </arm>
+</top>

```

Comment:
@fabferro these files are certainly not large, but in general this kind of data should go into the dedicated data repository
@smuzaffar or is this below the thresholds?